src/lib/api/collaboration-token.ts

```diff
@@ -1,6 +1,6 @@
 import useSWR from "swr";
 import { Jwt, parseJwt } from "@/types/jwt";
-import { sessionGuard } from "@/lib/session/session-guard";
+import { sessionGuard } from "@/lib/auth/session-guard";
```

Comment:
Curious about this change. What was the thinking here? (not opposed just curious)